_posts/2023/2023-03-13-assign-a-default-to-an-attribute-active-record.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Assign a default value to an attribute in Active Record"
+description: "You’ve probably been using callbacks. Don’t."
+layout: article
+category: ruby
+image:
+  base: '2023/assign-default-to-an-attribute-active-record'
+  alt: "Blank wood"
+  credit: "Keith Misner"
+  source: "https://unsplash.com/photos/h0Vxgz5tyXA"
+
+---
+
+If you ever needed to set a default value in an Active Record model, you probably used a callback.
+
+Since Rails 5.0 there’s [been a better way](https://edgeguides.rubyonrails.org/5_0_release_notes.html#active-record-attributes-api). I had missed it until recently (thanks [Moses](https://twitter.com/Gathukumose/status/1615298667031797760)!).
```

Comment:
```suggestion
Since Rails 5.0 there’s [been a better way](https://edgeguides.rubyonrails.org/5_0_release_notes.html#active-record-attributes-api). I had missed it until recently! (Thanks [Moses](https://twitter.com/Gathukumose/status/1615298667031797760)!)
```